tests/test_checks.py

```diff
@@ -1245,6 +1251,30 @@ def test_fail_isGreaterThanOrEqualTo(self):
         )
         self.assertEqual(self.isGreaterThanOrEqualTo("h", "f", lambda x: x == 1), [Row(constraint_status="Success")])
 
+    def test_where(self):
```

Comment:
Could you add a few more test cases?

1. on support checks (e.g. `hasCompleteness`)
2. continue chaining (e.g. add a `.satisfies`  after `.where `
3. Add/tweak this test case  https://github.com/awslabs/python-deequ/blob/e74e974e739f24cccb827a6378cd50e97697a0e8/tests/test_checks.py#L1285 to assert unsupported check

Reply:
Added